docs/platform/how-to/deploy-public-page.md

```diff
@@ -0,0 +1,49 @@
+# Public services
+
+The Quix SaaS platform allows you to deploy public facing web pages and API's.
```

Comment:
API's -> APIs.
public facing -> public-facing.
Can we extend the first paragraph to explain what this How-To will help the user achieve.